modules/installation-gcp-marketplace.adoc

```diff
@@ -4,45 +4,39 @@
 
 :_content-type: PROCEDURE
 [id="installation-gcp-marketplace_{context}"]
-= Using a GCP Marketplace image
-If you want to deploy an {product-title} cluster using a GCP Marketplace image, you must create the manifests and edit the compute machine set definitions to specify the GCP Marketplace image.
+= Using the GCP Marketplace offering
 
-.Prerequisites
-
-* You have the {product-title} installation program and the pull secret for your cluster.
+Using the GCP Marketplace offering lets you deploy an {product-title} cluster, which is billed on pay-per-use basis (hourly, per core) through GCP, while still being supported directly by Red Hat.
```

Comment:
Need a nonbreaking space between "Red" and "Hat?"
![image](https://github.com/openshift/openshift-docs/assets/83772007/16999161-cf47-4d8f-a59f-9b5d424150df)[
]
(https://redhat-documentation.github.io/supplementary-style-guide/#introduction)